wandb/cli/cli.py

```diff
@@ -650,7 +650,31 @@ def _clean():
 @click.option("--program", default=False, help="Set sweep program")
 @click.option("--settings", default=False, help="Set sweep settings", hidden=True)
 @click.option("--update", default=None, help="Update pending sweep")
-@click.argument("config_yaml")
+@click.option(
+    "--stop",
+    is_flag=True,
+    default=False,
+    help="Finish a sweep to stop running new runs and let currently running runs finish.",
+)
+@click.option(
+    "--cancel",
+    is_flag=True,
+    default=False,
+    help="Cancel a sweep to kill all running runs and stop running new runs.",
```

Comment:
```suggestion
    help="Cancel a sweep: kill all running runs and stop running new runs.",
```

Reply:
Cancel a sweep. This command terminates the sweep agent process and kills all currently running runs.